contexts/StakingContext.tsx

```diff
@@ -19,7 +20,8 @@ export interface StakingContextState {
   tokenAllowance: BigNumber | undefined;
   unstakeRequestTime: Date | undefined;
   canUnstakeTime: Date | undefined;
-  unstakeCoolDown: number | undefined;
+  unstakeCoolDown: BigNumber | undefined;
+  resetOutstandingRewards: () => void;
```

Comment:
I realized that I did not update the query updater logic to reset the outstanding rewards that we are calculating instead of querying